.vs/VSWorkspaceState.json

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
More Merge Conflict Things. This should be .gitignored

Reply:
Fixed